app/src/services/consts-service.js

```diff
@@ -33,6 +33,7 @@ angular.module('otServices')
                 CHEMBL: 'chembl',
                 DISGENET: 'disgenet',
                 EPMC: 'europepmc',
+                NFERX: 'nferx',
```

Comment:
Luca is already migrating this service to a config file so you will have to create your version (as you are currently doing with default.json vs custom.json)